CommunicationLayer/MessagingFramework/MpptMessage.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <QByteArray>
+#include <QString>
+
+class MpptMessage
+{
+public:
+    MpptMessage(const QByteArray& messageData);
+
+    unsigned char mpptStatus() const;
+    unsigned short arrayVoltage() const;
```

Comment:
same as comment far below, I interpreted the serial sheet as having bits 0-2 of the MpptStatus containing the MpptNumber